jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -112,6 +119,16 @@ private Probe discoverWildflyJARHealthCheck(boolean isReadiness) {
             String scheme = getScheme().toUpperCase();
             String path = isReadiness ? getReadinessPath() : getLivenessPath();
 
+            if (isStartup) {
+                int periodSeconds = getPeriodSeconds();
+                scheme = getScheme().toUpperCase();
+                path = getStartupPath();
+                return new ProbeBuilder()
+                        .withNewHttpGet().withNewPort(port).withPath(path).withScheme(scheme).endHttpGet()
+                        .withFailureThreshold(getFailureThreshold())
+                        .withSuccessThreshold(getSuccessThreshold())
+                        .withPeriodSeconds(periodSeconds).build();
```

Comment:
I see you've added a new periodSeconds config parameter. Why is it used only in case of startup probes?

I see both blocks for ProbeBuilder look the same. Can we change it like this?

```java
            int port = getPort();
            if (port <= 0) {
                return null;
            }
            // scheme must be in upper case in k8s
            String scheme = getScheme().toUpperCase();
            int periodSeconds = getPeriodSeconds();

            return new ProbeBuilder().withNewHttpGet()
                .withNewPort(port).withPath(getProbePath(isReadiness, isStartup)).withScheme(scheme)
                .endHttpGet()
                .withFailureThreshold(getFailureThreshold())
                .withSuccessThreshold(getSuccessThreshold())
                .withInitialDelaySeconds(getInitialDelaySeonds(isReadiness, isStartup))
                .withPeriodSeconds(periodSeconds)
                .build();
```

Reply:
Umm, from what I read in Kubernetes documentation `initialDelaySeconds` and `periodSeconds` are different fields. The former is wait period before performing the first probe and the latter is time interval between performing probes repeatedly.

Reply:
Sorry, I didn't mean to remove `periodSeconds` but use it in the case of all three probes.

Reply:
Sorry for the misunderstanding and thanks for the clarification.